tests/virtualization/universal/hotplugging.pm

```diff
@@ -216,6 +216,11 @@ sub run_test {
         save_screenshot;
     }
 
+    # 0. Guest preparation - Increase maximum memory for this test run
+    shutdown_guests();
+    assert_script_run("virsh setmaxmem $_ 4G --config && virsh start $_") foreach (keys %virt_autotest::common::guests);
```

Comment:
How about the guests with mem >=4G already?

Reply:
We don't have any SLES guests with >= 4G at the moment and given the memory constrains I doubt we will have them in the near future, so I fail to see the problem you'd like to solve.

Could you illustrate the problem you want to solve? Thinking about adding >4GB guests in the future or thinking about testing >4GB?

Reply:
This is not a problem for now definitely. But there will likely be such bigger mem guests in not far future. So suggest to make maxmem configurable from testsuite level, if not much effort needed.

Reply:
Ok there is a problem with making this configurable: We have [too many xen guests](https://progress.opensuse.org/issues/93576) at the moment, this is why we had to limit the maximum memory per guest from 4GB to 2 GB. In hotplugging however we increase the amount of memory to 4GB otherwise we cannot test the memory increase.

I'm gonna tackle this issue by increasing the maximum memory in `hotplugging` by 2GB additional to the maximum memory. This should give us the freedom to configure the maximum memory, while it also allows us to keep our guests with 2GB of ram.

Reply:
@alice-suse Please review my current suggestion and let me know, if this is what you had in mind.

Reply:
I assume the thumbsup means your comment has been addressed. I'm marking this as resolved, feel free to reopen if there are still concerns.